src/foam/nanos/cron/Schedulable.js

```diff
@@ -61,13 +110,123 @@
+      factory: function() {
+        if ( ! this.eventDaoKey ) return;
+
+        const E = foam.mlang.ExpressionsSingleton.create({});
+        var idPredicate = E.EQ(this.ScriptEvent.OWNER, this.id);
+        var datePredicate = E.LT(this.ScriptEvent.LAST_RUN, new Date());
+
+        return this.__subContext__[this.eventDaoKey].where(datePredicate)
+                                                    .where(idPredicate);
+      },
+      view: function(_, X) {
+        var dao = X.data.filteredEventDAO;
+        return {
+          class: 'foam.comics.v2.DAOBrowserView',
+          config: {
+            class: 'foam.comics.v2.DAOControllerConfig',
+            dao: dao,
+            summaryView: {
+              class: 'foam.u2.table.TableView',
+              columns: [ 'lastRun' ],
+              editColumnsEnabled: false,
+              disableUserSelection: true
+            }
+          }
+        };
+      }
+    },
     {
       class: 'String',
       name: 'objectDAOKey'
     },
     {
       class: 'FObjectProperty',
-      name: 'objectToSchedule'
+      name: 'objectToSchedule',
+      createVisibility: 'HIDDEN'
+    },
+    {
+      class: 'String',
+      name: 'name',
+      section: 'summary',
+      createVisibility: 'HIDDEN',
+      gridColumns: 4,
+      order: 1
+    },
+    {
+      class: 'Date',
+      name: 'nextScheduledDate',
+      section: 'summary',
+      createVisibility: 'HIDDEN',
+      gridColumns: 4,
+      order: 2,
+      javaFactory: `
+        Date d = getSchedule().getNextScheduledTime(getX(), new Date());
+        return d != null ? d : null;
+      `
+    },
+    {
+      class: 'foam.core.Enum',
+      of: 'foam.nanos.auth.LifecycleState',
+      name: 'lifeCycleState',
+      label: 'Status',
+      section: 'summary',
+      createVisibility: 'HIDDEN',
+      gridColumns: 4,
+      order: 3,
+      factory: function() {
+        if (
+          ! this.objectToSchedule ||
+          ! this.objectToSchedule.lifecycleState ||
+          ! this.objectToSchedule.lifecycleState.name
+        ) return;
+        return this.objectToSchedule.lifecycleState.name;
+      }
+    },
+    {
+      name: 'lastRun',
+      label: 'Last Occurrence'
+    },
+    {
+      class: 'UnitValue',
+      name: 'amount',
+      createVisibility: 'HIDDEN',
+      factory: function() {
+        if ( ! this.objectToSchedule || ! this.objectToSchedule.sourceAmount ) return;
+        return this.objectToSchedule.sourceAmount;
+      },
+      tableCellFormatter: function(amount, X) {
+        var formattedAmount = amount/100;
+        this
+          .add('$', X.addCommas(formattedAmount.toFixed(2)));
+      }
+    },
+    {
+      class: 'Enum',
+      of: 'foam.time.TimeUnit',
+      name: 'frequency',
+      createVisibility: 'HIDDEN',
+      factory: function() {
+        if ( ! this.schedule || ! this.schedule.frequency ) return;
+        return this.schedule.frequency;
+      }
+    },
+    {
+      class: 'Date',
```

Comment:
If you just set the view on the `schedule` prop to the simpleIntervalSchedule view can you not get rid of these props that just reference that property?